packages/ocaml/ocaml-base-compiler.4.14.1/opam

```diff
@@ -2,7 +2,10 @@ opam-version: "2.0"
 name: "ocaml-base-compiler"
```

Comment:
Aren't we on 4.14.2? Should we drop 4.14.1?

Reply:
Yes, eventually, some people might not have updated their opam switches to ocaml 4.14.2, which means bad things will happen when the version is removed.